src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task09.java

```diff
@@ -36,6 +36,21 @@ public static void main(String[] args) {
     static String convertToAccountingFormat(long amount) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String toAccountingFormat = "";
+        final int SEPARETION_THREE_SIGNS = 1000;
+        String minus = "";
+        if (amount < 0) {
+            minus = "-";
+            amount = -amount;
+        }
+        while (amount / SEPARETION_THREE_SIGNS != 0) {
+            if (amount % SEPARETION_THREE_SIGNS == 0) {
+                amount = amount / SEPARETION_THREE_SIGNS;
+                toAccountingFormat = " 000" + toAccountingFormat;
+            } else{
+                toAccountingFormat = " " + amount % SEPARETION_THREE_SIGNS + toAccountingFormat;
+            } amount = amount / SEPARETION_THREE_SIGNS;
+           }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"